src/renderer/qssWidget/js/qssSideCart.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * The grade that handles widget's sideCart content
```

Comment:
I think the name as supplied in the specs is actually "SideCar", not that it is terribly important : P

Reply:
Yup, you are right, all of the instances of "sideCart" including variable names, comments, and file names are renamed.